staking/slash/double-sign_test.go

```diff
@@ -0,0 +1,12 @@
+package slash
+
+import (
+	"testing"
+
+	"github.com/harmony-one/harmony/consensus/quorum"
+)
+
+func TestDidAnyoneDoubleSign(t *testing.T) {
+	d := quorum.NewDecider(quorum.SuperMajorityStake)
```

Comment:
how did you test it?

Reply:
Haven't, business logic hasn't been written yet. This was piping and infrastructure, next PR implements the double sign.